lc2fen.py

```diff
@@ -33,26 +33,35 @@ def parse_arguments():
     global ACTIVATE_KERAS, ACTIVATE_ONNX, ACTIVATE_TRT
 
     parser = argparse.ArgumentParser(
-        description="Predicts board configurations from images."
+        description="Predicts board configuration(s) (FEN string(s)) from image(s)."
     )
 
-    parser.add_argument("path", help="Path to the image or folder you wish to predict")
+    parser.add_argument(
+        "path", help="Path to the image or folder you wish to predict the FEN(s) for"
+    )
     parser.add_argument(
         "a1_pos",
-        help="Location of the a1 square in the chessboard "
+        help="Location of the a1 square in the chessboard image(s) "
         "(B = bottom, T = top, R = right, L = left)",
         choices=["BL", "BR", "TL", "TR"],
     )
+    parser.add_argument(
+        "previous_fen",
+        nargs="?",
+        help="FEN string of the previous board position (if "
+        "you are predicting the FEN for a single image and if "
+        "the previous board position is known)",
+    )
 
     inf_engine = parser.add_mutually_exclusive_group(required=True)
     inf_engine.add_argument(
-        "-k", "--keras", help="Run inference using Keras", action="store_true"
+        "-k", "--keras", help="run inference using Keras", action="store_true"
```

Comment:
The message for `-h` starts with the lowercase "s:"

<img width="864" alt="image" src="https://github.com/davidmallasen/LiveChess2FEN/assets/51491203/9588b527-1331-4ff0-acb4-6260e567c6c7">

I didn't if there's a way (or it's even a good idea) to change the message associated with `-h`, so I figured I might as well make the messages for the other arguments start with a lowercase letter as well.

Perhaps a case could even be made for changing "P" to "p" and changing "L" to "l" above for the positional arguments--if you'd like, I would push another commit for exactly that

Reply:
Wow, good catch. I hadn't realized that.
I would still keep the "P" and "L" since it allows to see more clearly the difference between the name of the argument and its description.